feature-libs/cart/import-export/assets/translations/en/import-export.i18n.ts

```diff
@@ -16,15 +16,16 @@ export const importEntriesDialog = {
   importProducts: 'Import Products',
   importProductsSubtitle:
     'Add products by importing a .CSV file and creating a new saved cart.',
-  importProductFileText:
-    'The text file should list the product SKUs and quantities, separated by a comma. File size should not exceed {{ size }} kb',
+  importProductFileDetails:
+    'The text file should list the product SKUs and quantities, separated by a comma.',
```

Comment:
What do you think about: 'Text file should contain list of products with required columns separated by comma: SKU and quantity'.